android-components/components/browser/icons/src/main/java/mozilla/components/browser/icons/BrowserIcons.kt

```diff
@@ -255,6 +262,47 @@ class BrowserIcons constructor(
         }
     }
 
+    @Composable
+    fun LoadIntoViewInternal(
```

Comment:
this fun is essentially a hybrid of the `loadIntoViewInternal` fun above that is used in non-Compose Top Sites and the `BrowserIcons.Loader` that was used to previously load the icon by FavIcon